src/common/integrations/protocolIntegrationPicked.ts

```diff
@@ -77,8 +78,22 @@ export const protocolIntegrationPicked = async (
       return;
     }
 
+    const entryId = options?.config_entry || entries![0].entry_id;
+    let longRangeSupported = false;
+
+    try {
+      const zwaveNetwork = await fetchZwaveNetworkStatus(hass, {
```

Comment:
Why doesn't the zwave js add node dialog query this?

Reply:
Yes good idea, I will fetch it if it's not there. So we can keep the data we get from the dashboard where the controller data is already fetched.